RMG/4Day/로또의 최고 순위와 최저 순위.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * 자기의 로또번호와 실제 당첨 번호를 맞추어 확인하는 함수입니다.
+ * 그러나 지워져서 안 보이는 번호는 0으로 저장하고, 최소와 최대를
+ * 맞추어봅니다. (배열의 순서는 상관없습니다.)
+ * solution([44, 1, 0, 0, 31, 25],[31, 10, 45, 1, 6, 19]) // [3, 5]  -> 3등 5등
+ * solution([0, 0, 0, 0, 0, 0], [38, 19, 20, 40, 15, 25]) // [1, 6] -> 1등 6등
+ * solution([45, 4, 35, 20, 3, 9], [20, 9, 3, 45, 4, 35]) // [1, 1] -> 1등 1등
+ */
+function solution(lottos, win_nums) {
+  let mathCount = 0; // 일치하는 번호의 개수를 저장하는 변수를 선언합니다.
+  let zeroCount = 0; // 0의 개수를 저장하는 변수를 선언합니다.
+
+  // lottos 배열을 순회하며 일치하는 번호와 0의 개수를 셉니다.
+  for (let i = 0; i < lottos.length; i++) {
+    if (lottos[i] === 0) {
+      zeroCount++; // 0이면 카운트 증가
+    } else {
+      for (let j = 0; j < win_nums.length; j++) {
+        if (lottos[i] === win_nums[j]) {
+          mathCount++; // 일치하는 번호가 있으면 카운트 증가
+        }
+      }
+    }
+  }
+
+  // 최고 순위와 최저 순위를 계산합니다.
+  let highestRank, lowestRank;
```

Comment:
```suggestion
  let highestRank = 0;
  let lowestRank = 0;
```
초기값을 지정해준다면 해당 변수가 어떤 데이터 타입을 갖는 변수인지 파악이 쉽습니다.

Reply:
값이 없어 보기 편하게 한번에 선언을 하려고 했었습니다...ㅠㅠ
앞으로는 선언할 때 유의해서 하겠습니다 !!